mooncake-integration/store/store_py.cpp

```diff
@@ -45,6 +45,10 @@ class MooncakeStorePyWrapper {
         return (store_ && (use_dummy_client_ || store_->client_));
     }
 
+    std::string get_tp_key_name(const std::string &base_key, int rank) {
+        return base_key + "_tp_" + std::to_string(rank);
```

Comment:
Currently, It is a workaround. Finally, we will embed this metadata within another data area.